README.md

```diff
@@ -1,2 +1,135 @@
 # probation-offender-events
 
+[![CircleCI](https://circleci.com/gh/ministryofjustice/probation-offender-events/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/probation-offender-events)
+[![Docker](https://quay.io/repository/hmpps/probation-offender-events/status)](https://quay.io/repository/hmpps/probation-offender-events/status)
+
+### Generate events for the offender changes in probation
+
+Background service for generating events for the **probation-events-topic** from Delius changes indicating a change to an offender in probation
```

Comment:
I think the topic display name is `probation-offender-events` - at least it is in the terraform.

Reply:
Thanks, corrected